### ISSUE TITLE
HA core update 2024.02.* -> Failed to fetch registers for group {xxx}

### ISSUE THREAD

**Issue by simowNgithub** [kodebach/hacs-idm-heatpump#88]
## Version of the custom_component

https://github.com/kodebach/hacs-idm-heatpump/releases/tag/v0.7.1

## Configuration

```yaml

no additional config in the yaml.

```

## Describe the bug
- Updated to HA core 2024.02.*
- IDM Integration throws errors below

## Debug log

```text

2024-02-19 09:29:33.544 ERROR (MainThread) [custom_components.idm_heatpump] Failed to fetch registers for group 1870 (count=5): Modbus Error: [Connection] Not connected[AsyncModbusTcpClient 10.10.10.94:502]
2024-02-19 09:29:33.544 ERROR (MainThread) [custom_components.idm_heatpump] Failed to fetch registers for group 4122 (count=2): Modbus Error: [Connection] Not connected[AsyncModbusTcpClient 10.10.10.94:502]
2024-02-19 09:30:03.360 ERROR (SyncWorker_31) [pymodbus.logging] Connection to (10.10.10.94, 502) failed: timed out
2024-02-19 09:30:06.364 ERROR (SyncWorker_51) [pymodbus.logging] Connection to (10.10.10.94, 502) failed: timed out

```

**Comment by simowNgithub** [comment]
solved, navigator 2.0 displayed old values and have to get a restart.

**Comment by kodebach** [comment]
Yeah, for anything that says "timed out", "not connected" or "Connection error" in general, I would recommend restarting everything including the devices on both ends (physical machine running Home Assistant & the heat pump) and any network stuff in between. This often fixes the issue (especially if WiFi is involved)
